src/scripts/main.js

```diff
@@ -1,7 +1,33 @@
 'use strict';
 
+const body = document.querySelector('body');
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.createElement('div');
+
+  setTimeout(() => {
+    element.style.display = 'none';
```

Comment:
you need to remove element from the page not just hide it with CSS property

Reply:
but test do not pass if i use element.remove()